configuration/modules.yaml

```diff
@@ -103,30 +103,49 @@ modules:
 
     blacklist: [
       '%gcc@4.8.5',
+      'atk',
       'autoconf',
       'automake',
       'boost~mpi',
       'cairo',
+      'cairo',
       'cmake',
       'curl',
       'flex',
       'freetype',
+      'gdk-pixbuf',
       'gettext',
+      'ghostscript',
+      'ghostscript-fonts',
       'glib',
+      'gobject-introspection',
+      'graphicsmagick',
+      'graphviz',
```

Comment:
no need to blacklist (contains usefull tools)

Reply:
were you thinking of graphicsmagick? graphviz is installed in the system (it's for the dot graph language), graphicsmagick is the one with the convert subcommands,etc